universe/interface.go

```diff
@@ -283,6 +288,19 @@ type IssuanceItem struct {
 	MetaReveal *proof.MetaReveal
 }
 
+// BatchRegistrar is an interface that allows a caller to register a batch of
+// issuance items within a base universe.
+type BatchRegistrar interface {
```

Comment:
I think we can also use this in the caretaker, so at `BatchStateConfirmed` when we go to insert all the items into the local tree.

Reply:
Definitely. We do that in a later commit.